pkg/plugins/policies/meshtrafficpermission/plugin/v1alpha1/plugin.go

```diff
@@ -74,7 +70,11 @@ func (p plugin) Apply(rs *core_xds.ResourceSet, ctx xds_context.Context, proxy *
 		}
 		rules, ok := mtp.FromRules.Rules[key]
 		if !ok {
-			continue
+			if len(proxy.Policies.TrafficPermissions) == 0 {
```

Comment:
It is not supported to mix TP and MTP. Let's not add custom code trying to do their own thing in each policy

Reply:
ok, but how do you want to add a default behavior on the lack of MTP? If we miss this check, then we will override any trafficpermission

Reply:
Ha you are right indeed.